mne/viz/evoked.py

```diff
@@ -1498,9 +1502,31 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
         parameters will be passed to the line plot call of the corresponding
         condition, overriding defaults.
         E.g., if evokeds is a dict with the keys "Aud/L", "Aud/R",
-        "Vis/L", "Vis/R", `styles` can be `{"Aud/L":{"linewidth":1}}` to set
+        "Vis/L", "Vis/R", `styles` can be `{"Aud/L": {"linewidth": 1}}` to set
         the linewidth for "Aud/L" to 1. Note that HED ('/'-separated) tags are
         not supported.
+    cmap : None | str | tuple
+        If not None, plot evoked activity with colors from a color gradient
+        (indicated by a str referencing a matplotlib colormap - e.g., "viridis"
+        or "Reds").
+        If ``evokeds`` is a list and ``colors`` is `None`, the color will
+        depend on the list position. If ``colors`` is a list, it must contain
+        integers where the list positions correspond to ``evokeds``, and the
+        value corresponds to the position on the colorbar.
```

Comment:
I am not sure this complexity is needed. If you want a different order of colors, you can reorder Evokeds, and when you need specific colors you can get these from the colormap yourself and pass to `colors`.

Reply:
Reordering Evokeds won't work for the dict case (which is my preferred case), and won't work for the important case where multiple evokeds get the same color.

Taking colors from an MPL (or, easier, seaborn) colormap is nice and all, but getting *everything*. set up - particularly the split legends - is hell on the user.

Reply:
Yes, after looking at how easy and elegant the example some lines below is - I tend to agree.
However as I mentioned in another comment it would be more flexible if colors could get floats between 0 and 1 and the linear order was not assumed.